core/node/da_clients/src/eigen/sdk.rs

```diff
@@ -39,14 +42,14 @@ pub(crate) const DATA_CHUNK_SIZE: usize = 32;
 pub(crate) const AVG_BLOCK_TIME: u64 = 12;
 
 impl RawEigenClient {
-    pub(crate) const BUFFER_SIZE: usize = 1000;
-
     pub async fn new(private_key: SecretKey, config: EigenConfig) -> anyhow::Result<Self> {
         let endpoint =
             Endpoint::from_str(config.disperser_rpc.as_str())?.tls_config(ClientTlsConfig::new())?;
-        let client = DisperserClient::connect(endpoint)
-            .await
-            .map_err(|e| anyhow::anyhow!("Failed to connect to Disperser server: {}", e))?;
+        let client = Arc::new(Mutex::new(
+            DisperserClient::connect(endpoint)
+                .await
+                .map_err(|e| anyhow::anyhow!("Failed to connect to Disperser server: {}", e))?,
```

Comment:
Why anyhow here?

Reply:
It's actually unneeded, I have removed it!